packages/broker/src/ConfigWizard.ts

```diff
@@ -138,54 +148,55 @@ const PLUGIN_NAMES: {[pluginName: string]: string} = {
     PUBLISH_HTTP: 'publishHttp'
 }
 
-const pluginSelectorPrompt = {
-    type: 'checkbox',
-    name:'selectPlugins',
-    message: 'Select the plugins to enable',
-    choices: Object.values(PLUGIN_NAMES)
-}
-
-const pluginPrompts: Array<inquirer.Question | inquirer.ListQuestion | inquirer.CheckboxQuestion> = []
-Object.keys(PLUGIN_DEFAULT_PORTS).map((pluginName) => {
-    const defaultPluginPort = PLUGIN_DEFAULT_PORTS[pluginName]
-    pluginPrompts.push({
-        type: 'input',
-        name: `${pluginName}Port`,
-        message: `Select a port for the ${pluginName} Plugin [Enter for default: ${defaultPluginPort}]`,
-        when: (answers: inquirer.Answers) => {
-            return answers.selectPlugins.includes(pluginName)
-        },
-        validate: (input: string | number): string | boolean => {
-            const portNumber = (typeof input === 'string') ? Number(input) : input
+const createPluginPrompts = (): Array<inquirer.Question | inquirer.ListQuestion | inquirer.CheckboxQuestion> => {
+    const selectPluginsPrompt: inquirer.CheckboxQuestion = {
```

Comment:
This method will be easier to read if we don't prefix most of the variables with word `plugin`. The context should be clear from the method name.